packages/accordions/demo/stories/StepperStory.tsx

```diff
@@ -21,7 +21,15 @@ export const StepperStory: Story<IArgs> = ({ steps, ...args }) => (
   <Stepper {...args}>
     {steps.map((step, index) => (
       <Stepper.Step key={index}>
-        <Stepper.Label icon={args.hasIcon && <Icon />} isHidden={args.isLabelHidden}>
+        <Stepper.Label
+          icon={args.hasIcon && <Icon />}
+          isHidden={args.isLabelHidden}
+          iconProps={{
+            role: 'img',
+            'aria-label': 'checked',
+            'aria-hidden': undefined
+          }}
```

Comment:
It would be good to initialize the default in the MDX `args` as a control (`args.iconProps`). That way the end user has full control for testing the rendered output.